ote_sdk/ote_sdk/entities/dataset_item.py

```diff
@@ -88,6 +92,9 @@ def __init__(
         roi: Optional[Annotation] = None,
         metadata: Optional[Sequence[MetadataItemEntity]] = None,
         subset: Subset = Subset.NONE,
+        ignored_labels: Optional[
```

Comment:
Good question, it is part of the SC backend design. Long story short, we do not create new AnnotationScenes upon label schema change due to performance and consistency reasons, but instead we store the revisit/ignore information in a parallel entity (AnnotationSceneState). When it comes to build the datasets, it's much easier and efficient for us to push this information in the DatasetItem rather than in the AnnotationScene.

Reply:
I see, thanks for explanation. I'm not familiar with the design proposal for this, but at a high level it seems that attaching ignored labels to the dataset item ROI, rather than a dataset item itself, could be a better choice in terms of consistency.